cypress-tests/cypress/support/pageBuilder/pbListBlockCategories.ts

```diff
@@ -24,11 +24,14 @@ const QUERY = /* GraphQL */ `
 
 Cypress.Commands.add("pbListBlockCategories", () => {
     cy.login().then(user => {
-        return gqlClient
-            .request({
-                query: QUERY,
-                authToken: user.idToken.jwtToken
-            })
-            .then(response => response.pageBuilder.listBlockCategories.data);
+        const client = new GraphQLClient(Cypress.env("GRAPHQL_API_URL"), {
```

Comment:
Something has changed here. It should be like the previous code, where we've used the `gqlClient`.